ci-operator/step-registry/openshift-qe/installer/bm/day2/netperf-external/openshift-qe-installer-bm-day2-netperf-external-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+
+# Setup Bastion
+  ## Change default gateway to Bastion host
+  ## Host Netserver at $EXTERNAL_SERVER_ADDRESS
+
+SSH_ARGS="-i ${CLUSTER_PROFILE_DIR}/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+bastion=$(cat ${CLUSTER_PROFILE_DIR}/address)
+
+ssh ${SSH_ARGS} root@${bastion}
+netserver -L 172.31.0.1
```

Comment:
This is OK with this for the testing, but `netserver` should be moved to a container. We cannot assume the bastion host has any software preinstalled.